website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -178,37 +177,45 @@ Separate from project and workspace permissions, you can grant teams permissions
 
 The following organization permissions are available:
 
-### Projects & workspaces permissions
+### Project permissions
 
-You must select a level of access for projects and workspaces.
+You must select a level of access for projects.
 
-#### Manage all projects & workspaces
+#### No access
```

Comment:
Should this be the "None" that is reflected on the setting page? All the other names seem to correspond directly to the name in the UI

Reply:
I initially had this heading as "None" for consistency between the UI and docs but upon rereading it, felt like "None" wasn't a very descriptive heading, even though it's nested under the org permissions section. This being said though, I definitely don't feel strongly about either heading and do like keeping things consistent, so I'm happy to defer to y'all! What do you, @emlanctot? 🤔

Reply:
I don't feel strongly, I think I would keep it consistent to the UI headers. But I would defer to education team !